build.sbt

```diff
@@ -23,7 +23,8 @@ def otaPlusProject(name: String): Project = Project(name, file(name))
       "org.scala-lang" % "scala-library" % scalaVersion.value,
       "org.scala-lang.modules" % "scala-xml_2.12" % "1.1.0",
       "org.scala-lang.modules" %% "scala-parser-combinators" % "1.0.4",
-      "com.google.guava" % "guava" % "18.0"
+      "com.google.guava" % "guava" % "18.0",
+      "com.advancedtelematic" %% "libats-logging" % "0.3.0-18-g104ed2d"
```

Comment:
this should be here https://github.com/advancedtelematic/ota-plus-server/blob/master/project/Dependencies.scala#L29 which then gets included here https://github.com/advancedtelematic/ota-plus-server/blob/ee7106a1f79d7cd3cf3a5fd86b953d691b1d41e5/ota-plus-web/build.sbt#L60

Reply:
Done.